oct/ansible/oct/roles/inventory/files/general.yml

```diff
@@ -8,4 +7,6 @@ openshift_master_identity_providers:
   - name: "allow_all"
     login: "true"
     challenge: "true"
-    kind: "AllowAllPasswordIdentityProvider"
\ No newline at end of file
+    kind: "AllowAllPasswordIdentityProvider"
+deployment_type: "origin"
+ansible_ssh_user: "ec2-user"
```

Comment:
We won't want this guy in here -- it lives in the `host_vars` for this inventory.